lib/ayesql.ex

```diff
@@ -131,6 +131,38 @@ defmodule AyeSQL do
     ]
   }
   ```
+  AyeSQL also allows you to choose the type of returned data structures.
+  Instead of the default map you can also pass an into option to your query
+  possible values are:
+  - Map or %{}
```

Comment:
```suggestion
  - An empty map or `%{}`
```

Reply:
Maybe something like that will be better:

  - an empty map: `Map.new()` or `%{}`                                                                                                         
  - an empty list: `Keyword.new()` or `[]`